packages/data-stores/src/domain-suggestions/queries.ts

```diff
@@ -70,7 +70,7 @@ export function useGetWordPressSubdomain( paidDomainName: string ) {
 		quantity: 1,
 		include_wordpressdotcom: true,
 		include_dotblogsubdomain: false,
-		only_wordpressdotcom: true,
+		only_wordpressdotcom: false,
```

Comment:
Could you help me understand why we’d need this change, and also if we could separate it into another PR if it's unrelated? 

From the code, if it needs only_wordpressdotcom to be set as false to return the correct dotcom subdomains, it needs to be fixed from the backend rather than the front-end. If that’s the case, the backend fix will then break the front-end if we ship this change. 🤔

Reply:
Right I have removed this above change for now so that we can pick this up in a separate PR.